.gitignore

```diff
@@ -245,8 +251,11 @@ __pycache__/
 #Kdevelop project files
 *.kdev4
 
-# xCode
-xcuserdata
+# Xcode
+xcuserdata/
+*.xcscmblueprint
+*.xccheckout
+*.xcodeproj/*
```

Comment:
Wouldn't `*.xcodeproj/` be enough?

Reply:
In the Xcode projects I have around it seems that way, although I'm on the most recent version which seems to behave a little better and I don't use Xcode VCS/checkout or blueprint features enough to be certain.

These were adapted from https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore while omitting the pre-Xcode-8 compat entries and the last section, which IMO is mostly needed in *actual* Xcode iOS projects. (I.e. where you want to share certain IDE stuff in your team, which shouldn't matter for us.)